pylabrobot/machines/machine.py

```diff
@@ -38,6 +38,7 @@ def __init__(
     backend: MachineBackend,
     category: Optional[str] = None,
     model: Optional[str] = None,
+    **_kwargs
```

Comment:
kwargs seems not needed here, but maybe an artifact of using mixins?

Reply:
When you have a bunch of subclasses with different args, they get passed up the MRO chain. This is the standard way to "ignore them"

Reply:
this feels like bad design. why is it necessary?

Reply:
I agree, it feels weird, but I believe it's just how python does things

Reply:
🤖 also agrees <img width="827" alt="image" src="https://github.com/user-attachments/assets/95117ef5-3084-47ee-abca-33d90e416340">